deku-p/src/core/bin/benchmark/deku_benchmark.ml

```diff
@@ -267,15 +267,47 @@ let benches : (module BENCH) list =
     (module Ledger_transfer);
   ]
 
-let formatter ~name ~item_message ~data =
+let format_to_print ~name ~item_message ~data =
   Format.sprintf "%s with %s: %.3f\n" name item_message data
 
-let writer ~data = Format.eprintf "%s%!" data
+let format_to_csv ~name ~item_message ~data =
+  Format.sprintf "%s, %s, %f\n" name item_message data
+
+let write_to_file ~data ~file =
+  let ( / ) = Eio.Path.( / ) in
+  let run dir =
+    Eio.Path.save ~append:true ~create:(`If_missing 0o600) (dir / file) data
+  in
+  Eio_main.run @@ fun env -> run (Eio.Stdenv.cwd env)
+
+let write_to_terminal ~data ~file =
+  let _ = file in
+  Format.eprintf "%s%!" data
+
+type params = {
+  domains : int; [@env "DEKU_DOMAINS"]
```

Comment:
```suggestion
  domains : int; [@env "DEKU_DOMAINS"][@default 16]
```

Reply:
Also please delete the code starting on line 8 that defines `domains`.

Reply:
Done, good catches.